jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UISelectAssert.java

```diff
@@ -23,167 +23,220 @@
  */
 public class UISelectAssert<A extends UISelectAssert, E extends ISelector> extends UIAssert<A, E>
         implements ITextAssert<A> {
-
+    
     @JDIAction("Assert that '{0}' option selected for '{name}'")
-    public A selected(Matcher<String> condition) {
-        jdiAssert(element.selected(), condition);
+    public A selected(Matcher<String> condition, String... messages) {
+        jdiAssert(element.selected(), condition, messages);
         return (A) this;
     }
+    
     @JDIAction("Assert that '{0}' option selected for '{name}'")
-    public A selected(String option) {
-        jdiAssert(element.selected(option), Matchers.is(true));
+    public A selected(String option, String... messages) {
+        jdiAssert(element.selected(option), Matchers.is(true), messages);
         return (A) this;
     }
+    
     @JDIAction("Assert that '{0}' option selected for '{name}'")
-    public A selected(int i) {
-        jdiAssert(element.selected(i), Matchers.is(true));
+    public A selected(int i, String... messages) {
+        jdiAssert(element.selected(i), Matchers.is(true), messages);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)